### PR TITLE
[Refactor](load) merge streamload BE and FE properties parsing

### PR DESCRIPTION
**Pull request by sollhui**
# Proposed changes

Issue Number: #18476 

## Problem summary

stream load support load data both from BE and FE.And both BE and FE maintain the code for parsing attributes, which can be merged only in FE and improve the maintainability of the code.

## verify property list

- column_separator
- columns
- where
- line_delimiter
- strict_mode
- partitions
- timeout
- timezone
- merge_type
- max_filter_ratio
- trim_double_quotes
- skip_lines
- function_column.sequence_col
- hidden_columns

## Checklist(Required)

* [ ] Does it affect the original behavior
* [ ] Has unit tests been added
* [ ] Has document been added or modified
* [ ] Does it need to update dependencies
* [ ] Is this PR support rollback (If NO, please explain WHY)

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 34.61 seconds
 stream load tsv:          430 seconds loaded 74807831229 Bytes, about 165 MB/s
 stream load json:         24 seconds loaded 2358488459 Bytes, about 93 MB/s
 stream load orc:          60 seconds loaded 1101869774 Bytes, about 17 MB/s
 stream load parquet:          30 seconds loaded 861443392 Bytes, about 27 MB/s
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230516024028_clickbench_pr_144164.html

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 34.18 seconds
 stream load tsv:          419 seconds loaded 74807831229 Bytes, about 170 MB/s
 stream load json:         22 seconds loaded 2358488459 Bytes, about 102 MB/s
 stream load orc:          59 seconds loaded 1101869774 Bytes, about 17 MB/s
 stream load parquet:          31 seconds loaded 861443392 Bytes, about 26 MB/s
 insert into select:          77.6 seconds inserted 10000000 Rows, about 128K ops/s
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230522123825_clickbench_pr_148446.html

File: fe/fe-core/src/main/java/org/apache/doris/analysis/StreamLoadStmt.java [hunk 1]
```diff
@@ -147,12 +141,10 @@ public Separator getLineDelimiter() {
         return lineDelimiter;
     }
 
-    @Override
```
Comment (CalvinKirs) [inline fe/fe-core/src/main/java/org/apache/doris/analysis/StreamLoadStmt.java:150]: Why is this removed?

File: fe/fe-core/src/main/java/org/apache/doris/analysis/StreamLoadStmt.java [hunk 2]
```diff
@@ -452,14 +434,26 @@ private void setLineDelimiter(String oriLineDelimiter) throws AnalysisException
...
+    @Override
+    public ResourceDesc getResourceDesc() {
+        return null;
+    }
+
+    @Override
+    public LoadType getLoadType() {
+        return null;
+    }
+}
```
Comment (CalvinKirs) [inline fe/fe-core/src/main/java/org/apache/doris/analysis/StreamLoadStmt.java:459]: What is the special effect here?

Comment (HHoflittlefish777) [reply fe/fe-core/src/main/java/org/apache/doris/analysis/StreamLoadStmt.java:459]: @CalvinKirs it implement insertStmt

File: fe/fe-core/src/main/java/org/apache/doris/analysis/StreamLoadStmt.java (hunk 1 above)
Comment (CalvinKirs) [reply fe/fe-core/src/main/java/org/apache/doris/analysis/StreamLoadStmt.java:150]: I mean, why did you remove the `@Override` annotation, although Java does not mandate the use of the `@Override` annotation, but we still strongly recommend using this annotation to clarify the code intent and improve code readability and maintainability. If the `@Override` annotation is not used when implementing the interface method, the compiler will not give warnings or error prompts when the method signature has typos or parameter type mismatches, which may lead to hard-to-find errors .

**Comment by dataroaring** [comment]
run buildall

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by HHoflittlefish777** [comment]
run p0

**Comment by HHoflittlefish777** [comment]
run p0

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by HHoflittlefish777** [comment]
run arm

**Comment by HHoflittlefish777** [comment]
run buildall

File: fe/fe-core/src/main/java/org/apache/doris/analysis/StreamLoadTask.java [hunk 3]
```diff
@@ -287,104 +305,104 @@ public static StreamLoadTask fromTStreamLoadPutRequest(TStreamLoadPutRequest req
...
+            if (properties.get(Property.TEMP_PARTITIONS) != null) {
+                partitions = new PartitionNames(true, Lists.newArrayList(partNames));
             } else {
                 partitions = new PartitionNames(false, Lists.newArrayList(partNames));
             }
         }
         switch (request.getFileType()) {
             case FILE_STREAM:
-            // fall through to case FILE_LOCAL
+                // fall through to case FILE_LOCAL
```
Comment (freemandealer) [inline fe/fe-core/src/main/java/org/apache/doris/analysis/StreamLoadTask.java:333]: unnecessary change

File: fe/fe-core/src/main/java/org/apache/doris/analysis/StreamLoadTask.java [hunk 4]
```diff
@@ -400,7 +418,7 @@ private void setColumnToColumnExpr(String columns) throws UserException {
             throw new AnalysisException("failed to parsing columns' header, maybe contain unsupported character");
         } catch (AnalysisException e) {
             LOG.warn("analyze columns' statement failed, sql={}, error={}",
-                     columnsSQL, parser.getErrorMsg(columnsSQL), e);
+                    columnsSQL, parser.getErrorMsg(columnsSQL), e);
```
Comment (freemandealer) [inline fe/fe-core/src/main/java/org/apache/doris/analysis/StreamLoadTask.java:421]: unnecessary change

**Comment by HHoflittlefish777** [comment]
run p0

**Comment by HHoflittlefish777** [comment]
run p0

**Comment by HHoflittlefish777** [comment]
run p0

**Comment by HHoflittlefish777** [comment]
run p0

**Comment by HHoflittlefish777** [comment]
run p0

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by HHoflittlefish777** [comment]
run p0

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by HHoflittlefish777** [comment]
run p0

**Comment by HHoflittlefish777** [comment]
run p0

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by HHoflittlefish777** [comment]
run p0

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 44.74 seconds
 stream load tsv:          457 seconds loaded 74807831229 Bytes, about 156 MB/s
 stream load json:         19 seconds loaded 2358488459 Bytes, about 118 MB/s
 stream load orc:          58 seconds loaded 1101869774 Bytes, about 18 MB/s
 stream load parquet:          29 seconds loaded 861443392 Bytes, about 28 MB/s
 insert into select:          69.2 seconds inserted 10000000 Rows, about 144K ops/s
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230623110259_clickbench_pr_166683.html

**Comment by HHoflittlefish777** [comment]
run p1

**Comment by HHoflittlefish777** [comment]
run p0

**Comment by HHoflittlefish777** [comment]
run p0

**Comment by HHoflittlefish777** [comment]
run p1

**Comment by HHoflittlefish777** [comment]
run p1

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 43.72 seconds
 stream load tsv:          453 seconds loaded 74807831229 Bytes, about 157 MB/s
 stream load json:         19 seconds loaded 2358488459 Bytes, about 118 MB/s
 stream load orc:          57 seconds loaded 1101869774 Bytes, about 18 MB/s
 stream load parquet:          29 seconds loaded 861443392 Bytes, about 28 MB/s
 insert into select:          68.5 seconds inserted 10000000 Rows, about 145K ops/s
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230630121210_clickbench_pr_170542.html

**Comment by HHoflittlefish777** [comment]
run p1

**Comment by HHoflittlefish777** [comment]
run p0

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 40.68 seconds
 stream load tsv:          467 seconds loaded 74807831229 Bytes, about 152 MB/s
 stream load json:         19 seconds loaded 2358488459 Bytes, about 118 MB/s
 stream load orc:          58 seconds loaded 1101869774 Bytes, about 18 MB/s
 stream load parquet:          29 seconds loaded 861443392 Bytes, about 28 MB/s
 insert into select:          66.6 seconds inserted 10000000 Rows, about 150K ops/s
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230701034931_clickbench_pr_170785.html

**Comment by HHoflittlefish777** [comment]
run p1

**Comment by HHoflittlefish777** [comment]
run p1

**Comment by HHoflittlefish777** [comment]
run p1

**Comment by HHoflittlefish777** [comment]
run p1

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 40.56 seconds
 stream load tsv:          452 seconds loaded 74807831229 Bytes, about 157 MB/s
 stream load json:         20 seconds loaded 2358488459 Bytes, about 112 MB/s
 stream load orc:          57 seconds loaded 1101869774 Bytes, about 18 MB/s
 stream load parquet:          28 seconds loaded 861443392 Bytes, about 29 MB/s
 insert into select:          70.5 seconds inserted 10000000 Rows, about 141K ops/s
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230702033452_clickbench_pr_170932.html

**Comment by HHoflittlefish777** [comment]
run p1

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 38.17 seconds
 stream load tsv:          458 seconds loaded 74807831229 Bytes, about 155 MB/s
 stream load json:         19 seconds loaded 2358488459 Bytes, about 118 MB/s
 stream load orc:          57 seconds loaded 1101869774 Bytes, about 18 MB/s
 stream load parquet:          28 seconds loaded 861443392 Bytes, about 29 MB/s
 insert into select:          68.8 seconds inserted 10000000 Rows, about 145K ops/s
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230702065804_clickbench_pr_170955.html

**Comment by HHoflittlefish777** [comment]
run fe

**Comment by HHoflittlefish777** [comment]
run fe

**Comment by HHoflittlefish777** [comment]
run p1

**Comment by HHoflittlefish777** [comment]
run fe ut

**Comment by HHoflittlefish777** [comment]
run fe-ut

**Comment by HHoflittlefish777** [comment]
run feut

**Comment by HHoflittlefish777** [comment]
run p1

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 40.98 seconds
 stream load tsv:          458 seconds loaded 74807831229 Bytes, about 155 MB/s
 stream load json:         21 seconds loaded 2358488459 Bytes, about 107 MB/s
 stream load orc:          56 seconds loaded 1101869774 Bytes, about 18 MB/s
 stream load parquet:          29 seconds loaded 861443392 Bytes, about 28 MB/s
 insert into select:          70.5 seconds inserted 10000000 Rows, about 141K ops/s
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230702095934_clickbench_pr_170979.html

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 40.7 seconds
 stream load tsv:          456 seconds loaded 74807831229 Bytes, about 156 MB/s
 stream load json:         20 seconds loaded 2358488459 Bytes, about 112 MB/s
 stream load orc:          57 seconds loaded 1101869774 Bytes, about 18 MB/s
 stream load parquet:          29 seconds loaded 861443392 Bytes, about 28 MB/s
 insert into select:          68.2 seconds inserted 10000000 Rows, about 146K ops/s
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230702130340_clickbench_pr_171006.html

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 43.99 seconds
 stream load tsv:          462 seconds loaded 74807831229 Bytes, about 154 MB/s
 stream load json:         22 seconds loaded 2358488459 Bytes, about 102 MB/s
 stream load orc:          56 seconds loaded 1101869774 Bytes, about 18 MB/s
 stream load parquet:          29 seconds loaded 861443392 Bytes, about 28 MB/s
 insert into select:          69.7 seconds inserted 10000000 Rows, about 143K ops/s
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230702154254_clickbench_pr_171033.html

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by HHoflittlefish777** [comment]
run clickbench

**Comment by HHoflittlefish777** [comment]
run feut

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 38.75 seconds
 stream load tsv:          462 seconds loaded 74807831229 Bytes, about 154 MB/s
 stream load json:         20 seconds loaded 2358488459 Bytes, about 112 MB/s
 stream load orc:          58 seconds loaded 1101869774 Bytes, about 18 MB/s
 stream load parquet:          29 seconds loaded 861443392 Bytes, about 28 MB/s
 insert into select:          70.2 seconds inserted 10000000 Rows, about 142K ops/s
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230703033248_clickbench_pr_171103.html

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 38.25 seconds
 stream load tsv:          455 seconds loaded 74807831229 Bytes, about 156 MB/s
 stream load json:         20 seconds loaded 2358488459 Bytes, about 112 MB/s
 stream load orc:          57 seconds loaded 1101869774 Bytes, about 18 MB/s
 stream load parquet:          29 seconds loaded 861443392 Bytes, about 28 MB/s
 insert into select:          69.1 seconds inserted 10000000 Rows, about 144K ops/s
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230703065115_clickbench_pr_171233.html

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 44.98 seconds
 stream load tsv:          473 seconds loaded 74807831229 Bytes, about 150 MB/s
 stream load json:         20 seconds loaded 2358488459 Bytes, about 112 MB/s
 stream load orc:          57 seconds loaded 1101869774 Bytes, about 18 MB/s
 stream load parquet:          28 seconds loaded 861443392 Bytes, about 29 MB/s
 insert into select:          70.4 seconds inserted 10000000 Rows, about 142K ops/s
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230703091418_clickbench_pr_171311.html

**Comment by HHoflittlefish777** [comment]
run p1

**Comment by HHoflittlefish777** [comment]
run p0

**Comment by HHoflittlefish777** [comment]
run p1

**Comment by HHoflittlefish777** [comment]
run p0

**Comment by HHoflittlefish777** [comment]
run p1

**Comment by HHoflittlefish777** [comment]
run p0

**Comment by HHoflittlefish777** [comment]
run p0

**Comment by HHoflittlefish777** [comment]
run p1

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by HHoflittlefish777** [comment]
run p0

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 37.34 seconds
 stream load tsv:          456 seconds loaded 74807831229 Bytes, about 156 MB/s
 stream load json:         20 seconds loaded 2358488459 Bytes, about 112 MB/s
 stream load orc:          57 seconds loaded 1101869774 Bytes, about 18 MB/s
 stream load parquet:          28 seconds loaded 861443392 Bytes, about 29 MB/s
 insert into select:          68.7 seconds inserted 10000000 Rows, about 145K ops/s
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230704080256_clickbench_pr_171915.html

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 45.25 seconds
 stream load tsv:          447 seconds loaded 74807831229 Bytes, about 159 MB/s
 stream load json:         20 seconds loaded 2358488459 Bytes, about 112 MB/s
 stream load orc:          56 seconds loaded 1101869774 Bytes, about 18 MB/s
 stream load parquet:          28 seconds loaded 861443392 Bytes, about 29 MB/s
 insert into select:          25.7 seconds inserted 10000000 Rows, about 389K ops/s
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230709124417_clickbench_pr_175213.html

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 54.06 seconds
 stream load tsv:          509 seconds loaded 74807831229 Bytes, about 140 MB/s
 stream load json:         20 seconds loaded 2358488459 Bytes, about 112 MB/s
 stream load orc:          65 seconds loaded 1101869774 Bytes, about 16 MB/s
 stream load parquet:          31 seconds loaded 861443392 Bytes, about 26 MB/s
 insert into select:          29.1 seconds inserted 10000000 Rows, about 343K ops/s
 storage size: 17170626341 Bytes
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230709205251_clickbench_pr_175206.html

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 50.55 seconds
 stream load tsv:          504 seconds loaded 74807831229 Bytes, about 141 MB/s
 stream load json:         20 seconds loaded 2358488459 Bytes, about 112 MB/s
 stream load orc:          64 seconds loaded 1101869774 Bytes, about 16 MB/s
 stream load parquet:          31 seconds loaded 861443392 Bytes, about 26 MB/s
 insert into select:          29.1 seconds inserted 10000000 Rows, about 343K ops/s
 storage size: 17170108131 Bytes
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230710103812_clickbench_pr_175421.html

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 46.62 seconds
 stream load tsv:          452 seconds loaded 74807831229 Bytes, about 157 MB/s
 stream load json:         21 seconds loaded 2358488459 Bytes, about 107 MB/s
 stream load orc:          56 seconds loaded 1101869774 Bytes, about 18 MB/s
 stream load parquet:          28 seconds loaded 861443392 Bytes, about 29 MB/s
 insert into select:          25.8 seconds inserted 10000000 Rows, about 387K ops/s
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230710030630_clickbench_pr_175422.html

**Comment by HHoflittlefish777** [comment]
run p0

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 48.93 seconds
 stream load tsv:          446 seconds loaded 74807831229 Bytes, about 159 MB/s
 stream load json:         19 seconds loaded 2358488459 Bytes, about 118 MB/s
 stream load orc:          57 seconds loaded 1101869774 Bytes, about 18 MB/s
 stream load parquet:          29 seconds loaded 861443392 Bytes, about 28 MB/s
 insert into select:          25.8 seconds inserted 10000000 Rows, about 387K ops/s
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230711040818_clickbench_pr_176205.html

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 54.64 seconds
 stream load tsv:          511 seconds loaded 74807831229 Bytes, about 139 MB/s
 stream load json:         20 seconds loaded 2358488459 Bytes, about 112 MB/s
 stream load orc:          65 seconds loaded 1101869774 Bytes, about 16 MB/s
 stream load parquet:          32 seconds loaded 861443392 Bytes, about 25 MB/s
 insert into select:          29.0 seconds inserted 10000000 Rows, about 344K ops/s
 storage size: 17162707879 Bytes
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20230714124558_clickbench_pr_178334.html

**Comment by HHoflittlefish777** [comment]
run buildall

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 53.07 seconds
 stream load tsv:          509 seconds loaded 74807831229 Bytes, about 140 MB/s
 stream load json:         20 seconds loaded 2358488459 Bytes, about 112 MB/s
 stream load orc:          65 seconds loaded 1101869774 Bytes, about 16 MB/s
 stream load parquet:          31 seconds loaded 861443392 Bytes, about 26 MB/s
 insert into select:          29.0 seconds inserted 10000000 Rows, about 344K ops/s
 storage size: 17170622602 Bytes

**Comment by hello-stephen** [comment]
(From new machine)TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 45.4 seconds
 stream load tsv:          505 seconds loaded 74807831229 Bytes, about 141 MB/s
 stream load json:         20 seconds loaded 2358488459 Bytes, about 112 MB/s
 stream load orc:          64 seconds loaded 1101869774 Bytes, about 16 MB/s
 stream load parquet:          32 seconds loaded 861443392 Bytes, about 25 MB/s
 insert into select:          28.9 seconds inserted 10000000 Rows, about 346K ops/s
 storage size: 17161866831 Bytes
